helpers/summary-helper.js

```diff
@@ -1,6 +1,18 @@
 import { textSummary } from '../lib/external/k6-summary.js';
+import { htmlReport } from '../lib/external/k6-html-report.js';
 
 export function handleSummary(data) {
+    if (__ENV.K6_HOSTENV === 'local') {
+        return {
+            'result.html': htmlReport(data),
+            stdout: textSummary(data, { indent: ' ', enableColors: true }),
+        };
+    } else {
+        return handleSummaryTesting(data);
+    }
+}
+
+function handleSummaryTesting(data) {
```

Comment:
Minor:
```suggestion
function _handleSummaryTesting(data) {
```
As I understand it is used in this class only